### ISSUE TITLE
[KYC Match / KYC Fill-in] OpenID Foundation (OIDF) collaboration 

### ISSUE THREAD

**Issue by ToshiWakayama-KDDI** [camaraproject/KnowYourCustomer#91]
**Problem description**

Based on the presentation about OIDF eKYC-IDA made on February 6th, is there any better way for collaboration / allgienment with OIDF eKYC-IDA specs, as CAMARA adopts OpenID Connect features.

(Spin off from Issue #65, item No.11, as per Action Item #13.03)

BR

**Comment by bhjelm** [comment]
I would agree with @AxelNennker that opening an issue in the [eKYC-IDA repository](https://bitbucket.org/openid/ekyc-ida/issues?status=new&status=open&status=submitted) would start the discussion in the working group and anyone is free to attend the working group meetings. 

In terms of ways for collaboration, I can work with the eKYC-IDA Chairs to coordinate specific times to address CAMARA related issues/topics to allow CAMARA members to participate in the discussion. When working with GSMA, we also arranged workshops (in-person and virtual) to discuss current open issues and solutions.

**Comment by ToshiWakayama-KDDI** [comment]
Closed this issue.
